.github/workflows/release.yaml

```diff
@@ -45,3 +45,43 @@ jobs:
           asset_path: manifests/okd/kubevirt-tekton-tasks-okd.yaml
           asset_name: kubevirt-tekton-tasks-okd.yaml
           asset_content_type: text/plain
+      - name: Update tekton tasks manifests
```

Comment:
is this what we want? Updating the manifests after uploading them?

Reply:
Idea is, when new tag is released, this will create a PR with regenerated manifests with newest tag. I wanted to avoid manual interaction as much as possible.

Reply:
Uploaded manifests (in github release) are already updated with newest version. This updates manifests in repo

Reply:
What I find a bit bothersome with this approach is that the repo is in inconsistent state until this PR merges, in contrast what the latest release points to. If you would build the images and create release on PR merge the only delay would be the automation compared to the manual merge.

But your approach is valid as well,  I will leave it to your judgement.

Reply:
I understand your concerns, but in this position (when we have deploy manifests in repo) we have only two options:
1) manual releases as they are now
2) automation, which will have to do a update PR after release. 
From my point I am fine with option number 2, because it will have really small impact on user (user can use manifests which are in github release and are up-to-date). And if user would like to use newest code, user will have to build it from source code, which means automation will replace all tags to latest.